apps/unified_connector/tasks.py

```diff
@@ -52,14 +52,16 @@ def trigger_connector_sources(max_execution_time, threshold, limit):
     processed_unified_connectors = set()
     for source in sources_qs.all()[:limit]:
         try:
-            UnifiedConnectorLeadHandler.process_unified_connector_source(source)
+            UnifiedConnectorLeadHandler.process_unified_connector_source(
+                source)
```

Comment:
Just a note, We don't use black for deep right now. Maybe we can talk further about this. But for now, could you disable in your editor?